purchase_request_procurement/tests/test_purchase_request_procurement.py

```diff
@@ -0,0 +1,72 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Eficent Business and IT Consulting Services S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp.tests import common
+from openerp import fields
+from openerp.exceptions import ValidationError
```

Comment:
use **from openerp.exceptions import Warning as UserError** to comply with CONTRIBUTING.md